prow/plugins/override/override.go

```diff
@@ -442,6 +485,37 @@ Only the following contexts were expected:
 		}
 		done.Insert(status.Context)
 	}
+
+	// We want to interate over the checkruns, create one with the same name and mark it as successful.
+	// Tide has logic to pick the best checkrun result
+
+	for _, checkrun := range checkruns.CheckRuns {
```

Comment:
Why do we override all checkruns that failed rather than just the ones the user requested us to override?

Reply:
i'll fix that with ` if overrides.Has(checkrun.Name)`